### ISSUE TITLE
Feature Request: specified dimension profiling

### ISSUE THREAD

**Issue by KatoNamus** [ZigTheHedge/jeargh#1]
Hi ZigTheHedge.

First off, than you very much for taking up this task, I'm sure I'm not the only who's missed JER's profiling but isn't able to do something about it themselves.

As for my feature request, I'm unable to complete a profile due to some kind of issue with the Star Wars planets addon for Ad-Astra (the swplanets:kamino dimension specifically). As soon as I have a chance, I'll collect the relevant logs and present it to that mod's developer(s).

For now though, I was hoping you could assist with adding an optional parameter to your mod's command so one can choose which dimension (or maybe multiple dimensions) to profile. Even a whitelist/blacklist config option would suffice if that's possible (as you're specifically stating the mod doesn't do that in the mod description page).
I believe this would be the better way to do it as other mods might also have issues in the future and giving us players and modpack creators the option to exclude them would help us a helluva lot.

Please and thank you in advance

**Comment by ZigTheHedge** [comment]
Hey! 
Yes, I have a plan to add whitelist/blacklist. And it could help, if you provide a crash-report.

**Comment by KatoNamus** [comment]
> Blacklist/whitelist planned:

Excellent, thank you.

> Crash-Report:

Yeah, sorry I didn't post that at first. I know it's needed and I guess I must've been too distracted to remember to post it.
Anyhow,  here's the [crash-report on patstebin](https://pastebin.com/d9nqtere). I usually add the debug.log and latest.log pastebins as well but my debug.log was ~50MB and pastebin just hung up so I rather zipped it and shared it along with a zipped latest.log and the CurseForge modpack export (for someone else but feel free to use it to troubleshoot) via [Dropbox](https://www.dropbox.com/sh/e1evcgupb2esj4p/AAC4TnYwc0J96LambJ6izWB6a?dl=0) (don't worry, you don't need a Dropbox account to get the shared files)

Also, as per my comment on the mod's comments page; after I disabled `SWPlanets-forge-1.19.2-1.0.6.4.jar` so I could at least complete the profiling, it does generate the `world-gen.json` file under \Config\ but the droprates aren't visible anywhere ingame in JEI.

Thank you for looking into this, I'm really excited at the prospect of having this function back in my modpack.

**Comment by ZigTheHedge** [comment]
Yeah, that's definitely crash on generation, and there's nothing I can do about it from my side.
As of world-gen.json, have you tried to restart the MC client before testing? If so, could you upload world-gen.json as well?

**Comment by KatoNamus** [comment]
World-gen.json uploaded, now also in the Dropbox folder.

**Comment by ZigTheHedge** [comment]
The strange thing is - your world-gen.json works in my pack.
https://imgur.com/yAhX2Qb
https://imgur.com/I2Kwwtu
I don't have every mod which generated in your world-gen installed, but Blue Skies and Ad Astra are there...

**Comment by KatoNamus** [comment]
Interesting. I'm wondering if it's one of the other JEI addons that might be blocking it somehow. I'll test a bit and give feedback. Thanks again for looking into it.

**Comment by ZigTheHedge** [comment]
I've just remembered: there's a config option to allow or deny DIY data in jeresources.toml and jeresources-common.toml:
`
diyData = true
`

**Comment by KatoNamus** [comment]
Yup, mine is set to True. Here is the complete file, let me know if anything else is different from yours, please.

> #Range: 1 ~ 4
> itemsPerColumn = 4
> #Range: 1 ~ 4
> itemsPerRow = 4
> diyData = true
> showDevData = false
> enchantsBlacklist = ["flimflam", "soulBound"]
> hiddenTabs = []
> dimensionsBlacklist = [-11]
> disableLootManagerReloading = false

Edit: I'm pasting with the linebreaks but GitHub seems to be removing them in the code  block. Quoting works better

**Comment by KatoNamus** [comment]
I've checked without other JEI addons (Mekanism Multiblock, IE Multiblock, etc) but its still not showing up. Looks like I'll have to go the binary method to find culprit mod. That's going to take a while so please feel free to close this ticket if you want to. I believe I can still comment in a closed ticket to give feedback on which is the guilty mod.

**Comment by KatoNamus** [comment]
Hi there, sorry for it taking sooooo long but I eventually had to binary search my mods for another problem and worked this one into the process. It's [Universal Enchants](https://www.curseforge.com/minecraft/mc-mods/universal-enchants-forge) that's blocking JEARGH's info from displaying. I haven't logged an issue there yet. Will do if you don't want to but I think you'll be able to give them more relevant information.
